docker-compose.yaml

```diff
@@ -1,7 +1,6 @@
-version: "3"
 services:
   llama-cpp:
-    image: logdetective/runtime:latest-cuda
+    image: quay.io/logdetective/runtime:0.3.3-cuda
```

Comment:
We're going to need to keep this in sync with every release; should we update the release scripting to do that?

Reply:
I dislike it too, but this is only a temporary measure. My next JIRA ticket is to fix the action that pushes the images. So once that is done, we will go back to `latest`.